server/utils/index.js

```diff
@@ -0,0 +1,17 @@
+const {
+  generalSchema,
+  cohortSchema,
+  projectSchema,
+  editCohortSchema,
+  studentSchema,
+  loginSchema,
+} = require('./validation');
+
+module.exports = {
```

Comment:
@MohammedAlghazali This seems good right now.  In the future, if you have a lot of other kinds of utils (things that are not schemas) you could organize this further by putting all of these schemas into a named export:
```
module.exports = {
  schemas: {
    generalSchema,
    cohortSchema,
    projectSchema,
    ...
  },
  otherUtilExport,
  ...
};
```

Reply:
yes, it will be very important to do that